README.md

```diff
@@ -141,6 +191,37 @@ All fields in the configuration file are optional:
 
       // Only used for "Fixed" timestamps.  Change this to generate more recent or more historic events.
       "at": "2022-02-01T01:01:01z"
+    },
+
+    // Required: Storage to sink generated events into
+    // Currently only a single output at a time is supported
+    "output": {
+      "file": {
+        // Generate files locally
+        "uri": "file:/path/to/my-events"
+        // Generate files into an S3 bucket
+        //"uri": "s3://my-bucket/my-events"
+      }
+      // "kafka": {
+        // Required: Seed brokers to use
+        // "brokers": "my-broker:9092,my-other-broker:9092",
+        // Required: Topic to deliver messages into
+        // "topic": "my-topic",
+        // Optional: Additional properties like authentication configuration
+        // "producerConfig": {
+        //  "additional-parameter": "value"
+        // }
+      // }
+      // "kinesis": {
+        // Required: Kinesis stream URI
+        // "uri": "kinesis://my-kinesis-stream",
```

Comment:
If I'm reading the code right, this example should say `streamName: my-kinesis-stream`, not uri.